logprep/abc/processor.py

```diff
@@ -8,7 +8,7 @@
 
 from attr import define, field, validators
 
-from logprep.framework.metric import Metric, calculate_new_average
+from logprep.framework.metrics.metric import Metric, calculate_new_average
```

Comment:
Could we eliminate the framework package?
Put it in logprep.metrics. It is not so much to write.

Reply:
Thx